test/unit_tests/braket/circuits/test_ascii_circuit_diagram.py

```diff
@@ -872,3 +872,35 @@ def __init__(self):
         "T  : |    0    |   1    |   2   |   3   |   4   |",
     )
     _assert_correct_diagram(circ, expected)
+
+
+def test_measure():
+    circ = Circuit().h(0).cnot(0, 1).measure([0])
+    expected = (
+        "T  : |0|1|2|",
+        "            ",
+        "q0 : -H-C-M-",
+        "        |   ",
+        "q1 : ---X---",
+        "",
+        "T  : |0|1|2|",
+    )
+    _assert_correct_diagram(circ, expected)
+
+
+def test_measure_multiple_targets():
+    circ = Circuit().h(0).cnot(0, 1).cnot(1, 2).cnot(2, 3).measure([0, 2, 3])
+    expected = (
+        "T  : |0|1|2|3|4|",
+        "                ",
+        "q0 : -H-C-----M-",
+        "        |       ",
+        "q1 : ---X-C-----",
+        "          |     ",
+        "q2 : -----X-C-M-",
+        "            |   ",
+        "q3 : -------X-M-",
+        "",
+        "T  : |0|1|2|3|4|",
+    )
+    _assert_correct_diagram(circ, expected)
```

Comment:
maybe we can also add a test that's a bit of an edge case? Like having a measure on q0 then a few more ops on q1 then a measure on q1?

Reply:
Added a test here and in `test_unicode_circuit_diagram` similar to what you suggested.